app/helpers/createAuthProvider.js

```diff
@@ -0,0 +1,15 @@
+const { AuthProvider, ErrorContext } = require('buying-catalogue-library');
+const { logger } = require('../logger');
+
+export const createAuthProvider = ({ config }) => {
+  const unauthenticatedError = new ErrorContext({
+    status: 401,
+    title: 'You\'re not authorised to view this page',
+    description: 'You must be logged in as a buyer to access Buying Catalogue orders.',
```

Comment:
this description need changed as it is admin not OF

Reply:
yes and this too.. yikes rushing it